libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -45,10 +47,37 @@ static int lua_micros(lua_State *L) {
     return 1;
 }
 
+static int lua_mission_receve(lua_State *L) {
+
```

Comment:
Should probably check that the argument count is correct here. (Should be 0). If we get the manual binding injection done correctly then it will be nice to move this to `mission:receive()`.

Reply:
added the check for 0 arguments, yeah it would be nice to move it over to the main table.